src/Components/Components/src/Forms/InputComponents/InputPassword.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Components.RenderTree;
+
+namespace Microsoft.AspNetCore.Components.Forms
+{
```

Comment:
This could be implemented as a `.razor` file now, but I'm keeping it as `.cs` for consistency with the others until we get round to updating them all.